definitions/spring-vm/vagrant.sh

```diff
@@ -19,6 +19,8 @@ mkdir -p "$chroot/root/.ssh" 2> /dev/null
 # add vagrant user
 chroot $chroot /bin/bash <<DATAEOF
 groupadd -r vagrant
+groupadd -r vboxsf
+groupadd -r vboxguest
```

Comment:
For some reason the bottom fails if the groups don't exist (e.g. on non-vbox), so here I'm creating them anyway, to make sure nothing breaks.